data-visualization/data-visualization.md

```diff
@@ -223,9 +223,14 @@ const g = d3.select("svg")
 
 ### Check for Understanding
 
-- Discuss how one would go about updating the DOM and utilizing D3.js. What are the general steps to set up a new D3.js visualization?
-- What factors do you consider when selecting a data set?
-- What do you consider when determining how to display data in a compelling way?
+- Which tools and open-source libraries are used for data visualization?
+- Which imports and methods from D3.js would you use to set up a new D3.js visualization or update the DOM?
+- What attributes are customizable for a chart or graph? List some.
+Form small groups and discuss:
```

Comment:
This doesn't actually end up on a new line in markdown.  Look at the markdown preview to make sure it looks as expected.